backend/src/Edna/DB/Schema.hs

```diff
@@ -0,0 +1,23 @@
+module Edna.DB.Schema
```

Comment:
I am reading the [tutorial](https://haskell-beam.github.io/beam/tutorials/tutorial1/) and I see that it starts with defining types that hold data from tables, like `ProjectT` in our case for example. Are they supposed to be defined here? Just checking, I think it's ok to do that after [EDNA-27](https://issues.serokell.io/issue/EDNA-27).

Reply:
>Are they supposed to be defined here?

Yes, in EDNA-38